src/keycloak/chart/templates/statefulset.yaml

```diff
@@ -200,17 +193,17 @@ spec:
               value: password
           {{- end }}
           {{- if eq (include "keycloak.postgresql.config" .) "true" }}
-            # Infinispan cache configuration
+            # Use Infinispan distributes caches
             - name: KC_CACHE
               value: ispn
-            - name: KC_CACHE_STACK
-              value: kubernetes
+            # Hints Infinispan to distribute data across machines in a Multi AZ environment
+            - name: KC_SPI_CACHE_EMBEDDED_DEFAULT_MACHINE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+                  apiVersion: v1
```

Comment:
Is there a good reference for what this env is/does? Mostly for my own understanding to understand how this works/helps with multi-AZ.

Reply:
The information about this is probably already published in the Keycloak nightly docs but in essence, this hints Infinispan to better distribute the data, see https://github.com/keycloak/keycloak/issues/41933

Reply:
Thanks, that's helpful!